### PR TITLE
Move everything to the LitaVersioner namespace

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

File: lib/lita_versioner/handlers/bumpbot_handler.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
+require "lita"
 require "forwardable"
+require_relative "../jenkins_http"
 
-module Lita
+module LitaVersioner
```
Comment (ksubrama) [inline lib/lita_versioner/handlers/bumpbot_handler.rb]: https://docs.lita.io/plugin-authoring/handlers/ recommends leaving handlers in Lita::Handlers.

At the very least, you'll need to change BumpbotHandler < Handler below to explicitly inherit from Lita::Handlers::Handler.

File: lib/lita_versioner/handlers/versioner.rb [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
 require "uri"
-require_relative "../project_repo"
+require_relative "../../lita_versioner/project_repo"
 require_relative "bumpbot_handler"
 
 module Lita
   module Handlers
     class Versioner < BumpbotHandler
 
+      # Common configuration must be defined once
```
Comment (ksubrama) [inline lib/lita_versioner/handlers/versioner.rb]: Why this?  Isn't this all defined in the base class?  Now we'd have 2 sets of config options.

File: lib/lita_versioner/handlers/bumpbot_handler.rb (hunk 1 above)
Comment (jkeiser) [inline lib/lita_versioner/handlers/bumpbot_handler.rb]: Hmm. OK, I'll move those back.

File: lib/lita_versioner/handlers/versioner.rb (hunk 2 above)
Comment (jkeiser) [inline lib/lita_versioner/handlers/versioner.rb]: Accidental. Will fix :)

**Comment by ksubrama** [comment]
:+1: assuming the travis gods are pleased.

**Comment by mwrock** [comment]
I'm :+1: once travis is happy. Looks like something got missed somewhere.

**Pull request merged**